### DISCUSSION TITLE
Configurator β (easepick 1.1.6)

### DISCUSSION THREAD

**Discussion by wakirin** [easepick/easepick#16] [Announcements]
Added [configurator](https://easepick.com/configurator).
In beta now.

<img width="767" alt="Screen Shot 2022-04-04 at 13 38 08" src="https://user-images.githubusercontent.com/7056055/161496106-b633a903-c9ce-4ee7-a5d5-6e44df78f9ef.png">

This might help better understand how some options work.

**Comment by DennisSuitters** [comment]
Nice, I was wondering how to configure the time plugin, as before there was no demo for it. I like this, easy, and looks great. Shame the timepicker doesn't look as nice as the rest, but I guess that can be worked on.

**Reply by wakirin** [reply]
> Nice, I was wondering how to configure the time plugin, as before there was no demo for it. I like this, easy, and looks great.

Thank you.

> Shame the timepicker doesn't look as nice as the rest, but I guess that can be worked on.

Yes, I agree. If you have an idea, feel free to suggest a new design (sketch) in a new discussion.
